packages/create-webpack-app/src/types.ts

```diff
@@ -0,0 +1,21 @@
+export type {
+  ActionConfig,
+  ActionType,
+  AddActionConfig,
+  AddManyActionConfig,
+  AppendActionConfig,
+  CustomActionFunction,
+  ModifyActionConfig,
+  PlopCfg,
+  PlopGenerator,
+  NodePlopAPI,
+  PlopGeneratorConfig,
+  Actions,
+} from "node-plop";
+
+export type InitOptions = { template: string; force?: boolean };
+export type LoaderOptions = { template: string };
```

Comment:
RE: slack conv, you should make this generic. You see the pattern

Reply:
Okay I will work on this soon, 
I will definitely use the already established types and interfaces in `@generators` package